Updated hooks to 4.10.4 (1000006)

Unbrick opened the pull request.

Unbrick committed:
Updated hooks to 4.10.3 (1000005) .

Unbrick committed:
Updated hooks to 4.10.4 (1000006) .

krokofant commented:
Thanks! :)

It seems you added duplicate hooks for the TimeView https://github.com/krokofant/JodelXposed/pull/7/commits/ac153a6cf0f67e06769520993ad04d82e1a4edd1#diff-f4ae5278eddd225956366a826a71d47eR336

Unbrick committed:
Updated hooks to 4.10.4 (1000006) .

Unbrick commented:
Whoops, wasnt my intention :P Fixed!

Unbrick committed:
- Updated hooks to 4.11.2 (1000009)
 - Added color change mod for posts
 - Implemented JodelXposed seamless in JodelApp (Swipe Right)
 - Removed SettingsActivity and its resources, not in use anymore
